src/app/webserver/authentication/anon-auth.component.ts

```diff
@@ -26,16 +27,26 @@ import { AuthenticationService } from './authentication.service';
         </div>
     `
 })
-export class AnonymousAuthenticationComponent implements OnDestroy {
+export class AnonymousAuthenticationComponent implements OnInit, OnDestroy {
     private _model: AnonymousAuthentication;
+    private _error: any;
     private _locked: boolean;
     private _original: AnonymousAuthentication;
     private _subscriptions: Array<Subscription> = [];
 
-    constructor(private _service: AuthenticationService) {
-        this._subscriptions.push(this._service.anonAuth.subscribe(auth => {
-            this.setFeature(auth);
-        }));
+    constructor(
+        private _service: AuthenticationService,
+        private _notificationService: NotificationService,
+    ) {
+    }
+
+    public ngOnInit() {
+        this._subscriptions.push(this._service.anonAuth.subscribe(
+            auth => this.setFeature(auth),
+            e => {
+                this._error = e;
+                this._notificationService.apiError(e);
```

Comment:
One issue here. NotificationService message is disappeared after timeout and users will lost the error message. Can we create a new notificationservice which does not have timeout?

Reply:
timeout will be removed in the next PR